WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2831,9 +2844,10 @@ public void onEditorFragmentInitialized() {
     }
 
     private void onEditorFinalTouchesBeforeShowing() {
-        refreshEditorContent();
+        refreshEditorTheme();
```

Comment:
Curious why `refreshEditorContent` was moved inside the gutenberg condition, and `refreshEditorTheme` was moved outside. Should these be reversed?

Reply:
Oh good catch that should be reversed not sure how that got switched up

Reply:
> Not sure what I was thinking when I got that switched up.

No sweat! One of the reasons for reviews :smile: